package.json

```diff
@@ -28,10 +28,9 @@
   "scripts": {
     "build": "rollup -c",
     "commit": "npx git-cz",
-    "lint": "tslint 'src/**/*.ts' 'test/**/*.ts' -t stylish",
-    "format": "prettier",
+    "lint": "tslint src/**/*.ts test/**/*.ts -t stylish --fix",
```

Comment:
actually, please revert :cry: Still a good idea to lint on CI to avoid when people didn't lint.

Reply:
if you want to lint fix manually you write `npm run lint -- --fix`

Reply:
but if linting is done on every commit, isnt that redundant ?

Reply:
no, cause I can commit from github where husky won't run :sweat_smile:

Reply:
True, ill add it back

Reply:
you'd want to remove the --fix since that will allow CI to pass :cry:

Reply:
yup lol just noticed. See the the latest commit

Reply:
hmm add prettier to your tslint config?

`npm i -D tslint-config-prettier prettier`
https://github.com/prettier/tslint-config-prettier

Reply:
Its already there

Reply:
:see_no_evil: